rio_tiler_pds/cbers/__init__.py

```diff
@@ -1,3 +1,4 @@
 """rio-tiler-pds.cbers"""
 
-from .utils import sceneid_parser  # noqa
+from rio_tiler_pds.cbers import aws  # noqa
+from rio_tiler_pds.cbers.utils import sceneid_parser  # noqa
```

Comment:
moving to absolute import resolves some circle dependencies error I was getting!